applications/crossbar/priv/couchdb/schemas/kapi.offnet_resource.req.json

```diff
@@ -77,6 +77,9 @@
         "Custom-SIP-Headers": {
             "type": "object"
         },
+        "Denied-Call-Restrictions": {
```

Comment:
Are they really a string? Or should this be an object? You can hint at it in the kapi module using a validator or you can manually correct this schema.